src/utils.js

```diff
@@ -256,3 +256,25 @@ export const isValidDate = (date) => {
 
   return Boolean(formattedValue.length <= 10);
 };
+
+/**
+ * Create a correct inner path depend on config PUBLIC_PATH.
+ * @param {string} checkPath - the internal route path that is validated
+ * @returns {string} - the correct internal route path
+ */
+export const createCorrectInternalRoute = (checkPath) => {
+  let basePath = getPath(getConfig().PUBLIC_PATH);
+
+  // Remove the trailing '/'
```

Comment:
[nit] I think this is an unnecessary comment

Reply:
Agree, removed